src/Microsoft.TestPlatform.PlatformAbstractions/uap10.0/Tracing/PlatformEqtTrace.cs

```diff
@@ -64,7 +64,8 @@ public void WriteLine(PlatformTraceLevel level, string message)
         /// <inheritdoc/>
         public bool InitializeVerboseTrace(string customLogFile)
         {
-            LogFile = Path.GetTempPath() + Path.GetFileNameWithoutExtension(customLogFile).Replace(" ", "_") + ".TpTrace.log";
+            // Generate log file name using a GUID, as Path.GetFileNameWithoutExtension is throwing exception
```

Comment:
Should we use string operation to get the name ?
This can be a fallback if the above fails, thoughts ?

Reply:
I'm waiting for UWP team to come back & tell us why the API GetFileNameWithoutExtension is failing in local machine scenario.
I don't want to write any extra logic till we have to.

Let me know if there are any concerns.